ge/views_utils.py

```diff
@@ -856,6 +857,39 @@ def create_excel_output(rpt_type: str) -> Workbook:
             # Projected Annual Income col is Q on non-UL endowments reports
             endowments_ws["Q3"] = as_of
 
+        # add border formatting to reports
```

Comment:
@ztucker4 If I'm reading this right, the code to set borders for endowments and gifts is exactly the same, just acting on different worksheets.  If so, this code should be in a separate function; pass in the worksheet, change it and return it.

If there is a difference between endowments and gifts here, it probably still can be a separate function, with a parameter to specify which to apply.